### PR TITLE
Setting up the Roger security.

### PR DESCRIPTION
**Pull request by benja-M-1**
Tests statuses: http://travis-ci.org/#!/benja-M-1/TheodoRogerCmsBundle/builds/1455949

### CODE REVIEW

**Comment by benja-M-1** [comment]
hey @marekkalnik can you give me some feedback on this? 

Thanks :)

File: Tests/Fixtures/app/AppTestKernel.php [hunk 1]
```diff
@@ -44,6 +44,9 @@ public function registerBundles()
             new \Symfony\Bundle\SecurityBundle\SecurityBundle(),
             new \Symfony\Bundle\TwigBundle\TwigBundle(),
             new \Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
+            new \JMS\AopBundle\JMSAopBundle(),
+            new \JMS\DiExtraBundle\JMSDiExtraBundle($this),
+            new \JMS\SecurityExtraBundle\JMSSecurityExtraBundle(),
             new \Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(),
```
Comment (marekkalnik) [inline Tests/Fixtures/app/AppTestKernel.php:47]: Are those JMS bundles really necessary? Is it not possible to handle the roles just with symfony core?

Comment (benja-M-1) [inline Tests/Fixtures/app/AppTestKernel.php:47]: I don't think so. At least the security is not necessary.

Comment (marekkalnik) [inline Tests/Fixtures/app/AppTestKernel.php:47]: Ok, so we'll merge and we'll look for other solution when we'll have some more time.

**Comment by marekkalnik** [comment]
Could you also add a little mention in the documentation on the roles you defined?

**Comment by benja-M-1** [comment]
Yes but working on it yesturday I realized that the groups that I defined are incomplete and well chosen.

File: Tests/Fixtures/app/AppTestKernel.php (hunk 1 above)
Comment (benja-M-1) [inline Tests/Fixtures/app/AppTestKernel.php:47]: I did it actually see 39f5a22 in this PR :)

**Pull request merged**